ls1intum/Artemis
# `Programming exercises`: Support python exercises for local continuous integration

### Checklist
#### General


#### Server

#### Changes affecting Programming Exercises

### Motivation and Context


With this PR, the local CI system supports Python exercises.

### Description

I added Python options for the build script generation. Test results get parsed accordingly.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- Local VC CI test server (3 or 6 (GitHub), 11 (old server))

1. As instructor, create a programming exercise with Python
2. Template and solution build result should be as expected
3. As student, create submission for this exercise
4. Build results should be as expected

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots
